cloud-operations/packer-image-builder/main.tf

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  compute_subnet_name = "image-builder"
+  compute_zone        = "${var.region}-a"
+}
+
+module "project" {
+  source          = "../../modules/project"
+  name            = var.project_id
+  parent          = try(var.root_node, null)
+  billing_account = try(var.billing_account, null)
```

Comment:
No need for `try()` here. `var.root_node` and `var.billing_account` are always defined even if they're null (i.e. the user doesn't give a value)

Reply:
I agree, fixed.